### PR TITLE
Orchestrator: Break resources into dependency trees

### PR DESCRIPTION
**Pull request by bnchrch**
## What
Theory: We are bundling too many resources on every sensor run, this reduces the resources we load to only what is required

### CODE REVIEW

**Review by evantahler — APPROVED** [review]
Makes sense... give it a shot!

File: airbyte-ci/connectors/metadata_service/orchestrator/.env.template [hunk 1]
```diff
@@ -1,4 +1,7 @@
-METADATA_BUCKET="ben-ab-test-bucket"
+METADATA_BUCKET="dev-airbyte-cloud-connector-metadata-service"
 CI_REPORT_BUCKET="airbyte-ci-reports"
 GITHUB_METADATA_SERVICE_TOKEN=""
 NIGHTLY_REPORT_SLACK_WEBHOOK_URL=""
+# METADATA_CDN_BASE_URL="https://connectors.airbyte.com/files"
+DOCKER_HUB_USERNAME=""
+DOCKER_HUB_PASSWORD=""
```
Comment (erohmensing) [inline airbyte-ci/connectors/metadata_service/orchestrator/.env.template:7]: Did you mean to commit these changes?

**Review by erohmensing — APPROVED** [review]
Reforestation! 🌲 🌲 🌲

File: airbyte-ci/connectors/metadata_service/orchestrator/.env.template (hunk 1 above)
Comment (bnchrch) [reply airbyte-ci/connectors/metadata_service/orchestrator/.env.template:7]: Yup!

**Pull request merged**